pkg/compare/referenceV1.go

```diff
@@ -204,6 +214,7 @@ func (config ReferenceTemplateConfigV1) GetFieldsToOmitRefs() []string {
 type ReferenceTemplateV1 struct {
 	*template.Template `json:"-"`
 	Path               string                    `json:"path"`
+	Description        string                    `json:"description,omitempty"`
```

Comment:
I think this should be on ReferenceTemplateV2. And the V1 template should just return an empty string.

Reply:
I think it's "safe" to add new optional fields to older APIs though, and V1 could benefit from this as well at no extra cost.

Reply:
Yeah shouldn't be an issue I suppose. 
Could you add a test for it then just to make sure we don't break `getMissingCRs` for anyone who decided to go with V1 and use the new field.

Reply:
Suddenly "no extra cost" became "adding another test", so I'm now less convinced ;)

Reply:
Test added.